release/nightly_tests/dataset/read_and_consume_benchmark.py

```diff
@@ -58,6 +62,11 @@ def consume_fn(ds):
             for _ in ds.iter_internal_ref_bundles():
                 pass
 
+    elif args.write:
+
+        def consume_fn(ds):
+            ds.write_parquet(WRITE_PATH)
```

Comment:
We can probably append a UUID to write path and delete it, so we can run multiple copies of this script.

Reply:
Added a random prefix to the write path.

The S3 lifecycle policy is configured to delete objects everday.